test/loader.spec.ts

```diff
@@ -0,0 +1,15 @@
+import 'mocha';
+import { assert } from 'chai';
+import { Test } from 'mocha';
+
+describe('Test Suite - web3 sdk', () => {
+  describe('Test case - Massa Wallets', () => {
+    it('Serialize & Deserialize.', () => {
+      assert(true);
+    });
+
+    it('Send a basic payload.', async () => {
+      assert(true);
```

Comment:
just to report that we don't have any unit test for now. When and how do we plan to add unit and test coverage KPI?

Reply:
I will add once the standard is approved by you and Greg.